wordpress-mobile/WordPress-iOS
# What's New - Remote announcements service

Ref #14584
WordPressKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/276/

Adds remote announcements service to read feature announcements from the backend

**Note: at the moment, the feature points to a specific announcement on the backend, since the database it's not yet populated for WPiOS. You will see a call to `getAnnouncements` in `AnnouncementsDataSource` that for now has hardcoded parameters to accomplish that**


To test:

- Go to Me/App Settings/What's New in WordPress
- Make sure that there's one announcement shown and has the same content (title, subtitle and image) as the screenshots below
- Try different devices/iOS versions and orientations and make sure that the views look right

iPhone | iPad
--- | ---
<img height="400" src="https://user-images.githubusercontent.com/34376330/91931409-fce8d680-eca8-11ea-8e25-06b515721051.png"> | <img height="400" src="https://user-images.githubusercontent.com/34376330/91931794-0c1c5400-ecaa-11ea-9a83-49b90fc8435e.png">

![IMG_0058]()


PR submission checklist:

- [x] ~~I have considered adding unit tests where possible.~~ Future PR
- [x] ~~I have considered adding accessibility improvements for my changes.~~ Future PR
